worker/src/handles/UnixStreamSocket.cpp

```diff
@@ -4,6 +4,7 @@
  */
 
 #define MS_CLASS "UnixStreamSocket"
+#define SOCKET_NAME "/tmp/TestSocket"
```

Comment:
Is SOCKET_NAME used anywhere, or should it go away?

Reply:
I need to remove this. I added this before pid requirement.